source/d3d8types.hpp

```diff
@@ -5,6 +5,7 @@
 
 #pragma once
 
+struct IUnknown;
```

Comment:
It doesn't - with `#include <unknwn.h>` either above or below `d3d9.h` the error is same.
Apparently it's a pretty well known issue with Clang compilation so far and people are working this around that way with success. I myself have used this workaround in the past without any side effects.